documentation/docs/05-changelog.md

```diff
@@ -3,6 +3,7 @@
 *??/??/2018*
 
 ##### New Features
+- Add `Location Prefabs` section to `AbstractMap`. Enables users to spawn prefabs at points-of-interest on the map directly from the `AbstractMap` UI.
 - Add `OnUpdated` event to `AbstractMap`. Enables subscribers to get a notification when the map location and/or zoom gets updated.
```

Comment:
Please make the changelog more detailed. Add a list of major features that have been added.